### PR TITLE
Project: Library app

### PR DESCRIPTION
**Pull request by Anna-Myzukina**
We created a small Library app.
You can add a new book.
Our book objects have:
- the book’s title, 
- author, 
- the number of pages, 
- and whether or not you have read the book

Please review our project!

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+        numOfPages: 335,
+        read: "Read" },
+        {title:"Think and Grow Rich",
+        author: "Napolean Hill",
+        numOfPages: 256,
+        read: "Unread" },]; 
+    }
+    else {
+    myLibrary = localStorage.getObj("library");
+}
```
Comment (maelfosso) [inline app.js]: Please, make this code lisible by indenting it.

File: app.js [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
+Storage.prototype.setObj = function(key, obj) {
+    return this.setItem(key, JSON.stringify(obj))
+}
+Storage.prototype.getObj = function(key) {
+    return JSON.parse(this.getItem(key))
+}
+
+if (storageAvailable('localStorage')) {
+    alert("Local Storage will be used");
```
Comment (maelfosso) [inline app.js]: That is not needed, please remove it.

**Review by maelfosso — CHANGES_REQUESTED** [review]
# Changes required :red_circle:

Hello coders,
Your project is OK. However, there is a requirement you forgot.
According to [that](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/library/new_book_button_requirement.png), you should make the form appear after clicking on `Add Book` button.

- Please, remove the `console.log` instructions.

- Also, please read the comment below.

Kindly make the suggested changes and submit a new code review request.
Thanks

Mael FOSSO

**Comment by emmanuelkamala** [comment]
1. I have added functionality whereby when a user clicks the 'New Book', then the form appears, so the form shows when a user clicks the 'New Book' button

2. I have removed all areas that had console.log in the app

**Review by jofer86 — APPROVED** [review]
## :wave: :wave: Hello!, I'm Jorge, and  I'm here to review this Pull Request, let's take a look shall we :eye: :eye: :eye: 
- Changes marked as **[OPTIONAL]** are not hard requirements but are nice to have.
- From what I can see, you have tackled all of the required changes asked by the previous reviewer!!!. This project is now complete. Great Job!!!
--------------------------------------------------------------------
Cheers, 
Jorge Fernando Rincón Figueroa.
Technical Support Engineer.
**Slack: @jofer86**

**Pull request merged**